plone/app/registry/exportimport/handler.py

```diff
@@ -26,29 +28,20 @@ def evaluateCondition(expression):
 
     ``expression`` is a string of the form "verb arguments".
 
-    Currently the supported verbs are ``installed`` and ``not-installed``.
-    """
-    arguments = expression.split(None)
-    verb = arguments.pop(0)
-
-    if verb not in ('installed', 'not-installed'):
-        raise ValueError("Invalid import condition: %r" % expression)
```

Comment:
@gforcada The opposite. It now re-uses the code from zope.configuration and all verbs in zope.configuration are supported (installed, not-installed, have, not-have).